x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -82,7 +83,7 @@ export const useEndpointActionItems = (
 
       const isolationActions = [];
 
-      if (isIsolated) {
+      if (isIsolated && canUnIsolateHost) {
         // Un-isolate is always available to users regardless of license level
```

Comment:
Should we update this comment if there is a permission now for un-isolate action?

Reply:
sure, we can update the comment. we should be keeping the check.